examples/tv-casting-app/tv-casting-common/core/CastingPlayer.cpp

```diff
@@ -29,7 +29,8 @@ namespace core {
 
 CastingPlayer * CastingPlayer::mTargetCastingPlayer = nullptr;
 
-void CastingPlayer::VerifyOrEstablishConnection(ConnectCallback onCompleted, unsigned long long int commissioningWindowTimeoutSec,
+void CastingPlayer::VerifyOrEstablishConnection(ConnectionCallbacks connectionCallbacks,
+                                                unsigned long long int commissioningWindowTimeoutSec,
```

Comment:
this should be an explicit size instead of this long size.... uint64_t (but again... why do we need so many seconds? I understand ms, but not seconds).

Reply:
Leaving a TODO for this.